src/main/java/me/zeroeightsix/kami/module/modules/movement/ElytraFlight.kt

```diff
@@ -184,6 +186,17 @@ class ElytraFlight : Module() {
         }
         event.cancel()
     })
+
+    private fun lookBoost() {
+        val readyToBoost = mc.player.movementInput.moveForward > 0 && ((mc.player.movementInput.jump && spaceBarTrigger.value) || !spaceBarTrigger.value) && mc.player.rotationPitch <= -10
+        val shouldAutoBoost = if (autoBoost.value) {
+            (mc.player.motionY >= (-fallSpeedControl.value) && sqrt(mc.player.motionX * mc.player.motionX + mc.player.motionZ * mc.player.motionZ) >= 0.8) || (mc.player.motionY >= 1) /*Make auto boost works great at all pitch*/
```

Comment:
this could also be written as `val shouldAutoBoost = (!autoBoost.value || ((mc.player.motionY >= (-fallSpeedControl.value) && sqrt(mc.player.motionX * mc.player.motionX + mc.player.motionZ * mc.player.motionZ) >= 0.8) || (mc.player.motionY >= 1))`